Lib/test/test_ctypes/test_aligned_structures.py

```diff
@@ -5,9 +5,9 @@
 )
 import struct
 import unittest
+from ._support import StructCheckMixin
 
-
-class TestAlignedStructures(unittest.TestCase):
+class TestAlignedStructures(unittest.TestCase, StructCheckMixin):
```

Comment:
I am not entirely sure but shouldn't mixin classes appear before non-mixin ones?